scripts/jenkins-yoctobuild-build.sh

```diff
@@ -297,10 +297,12 @@ if grep mender_servers <<<"$JOB_BASE_NAME"; then
                 if ! grep "COPY --from=build" Dockerfile; then
                     gulp build
                 fi
+                # GIT_REF + GIT_COMMIT for old versions, GIT_COMMIT_TAG for newer
                 docker build \
                     -t $docker_url:pr \
                     --build-arg GIT_REF=$(git describe) \
                     --build-arg GIT_COMMIT=$(git rev-parse --short HEAD) \
+                    --build-arg GIT_COMMIT_TAG="$(git describe)_$(git rev-parse --short HEAD)" \
```

Comment:
I think this is redundant. `git describe` already includes the commit SHA if the version isn't an exact tag.

Reply:
I agree. However this is the way that is build in its individual pipeline, ref https://github.com/mendersoftware/mendertesting/blob/master/.gitlab-ci-check-docker-build.yml#L72

@mzedel Do you have an opinion about it?

Reply:
I would be happy with `git describe` as well, since we should now never run this on an untagged gui repo...